.changeset/funny-shrimps-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': patch
+---
+
+Refactored the stepper component to handle `updateArgs` directly within the `getStepperItem` function, fixing the issue where stepper examples were not independent and resolving unreliable step changes that sometimes caused clicks to be unrecognized or froze the page.
```

Comment:
```suggestion
Fixed the issue where stepper examples were not independent and resolved unreliable step changes that sometimes caused clicks to be unrecognized or froze the page.
```

Reply:
Won't change this, because it's exactly how it was merged to the `main` branch.